README.md

```diff
@@ -1 +1,85 @@
+
+The service integrates with a number of internal and external systems. This includes [Transactions API](https://github.com/companieshouse/transactions.api.ch.gov.uk) and [api.ch.gov.uk](https://github.com/companieshouse/api.ch.gov.uk)
+
+Requirements
+------------
+
+To build psc-filing-api, you will need:
+* [Git](https://git-scm.com/downloads)
+* [Java 11](https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html)
+* [Maven](https://maven.apache.org/download.cgi)
+* [MongoDB](https://www.mongodb.com/)
+* Internal Companies House core services
+
+You will also need a REST client (e.g. Postman or cURL) if you want to interact with any psc-filing-api service endpoints.
+
+## Building and Running Locally
+
+1. From the command line, in the same folder as the Makefile run `make clean build`
+1. Checkout git submodule `api-enumerations`
+   1. `git submodule init` # initialize local configuration file
+   2. `git submodule update` # fetch all the data
+1. Configure project environment variables where necessary (see below).
+1. Ensure dependent Companies House services are running within the Companies House developer environment
+1. Start the service in the CHS developer environment
+1. Send a GET request using your REST client to persons-with-significant-control/healthcheck. The response should be 200 OK with status=UP.
+1. A database named `transaction_pscs` and the following collections are required:
+
+Collection name| Description                                                         | Data                                                                                                                                           
+--------------------|---------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+psc_submissions| the filing resource including id, dates, PSC id and etag, and links | this and the database will be created by the service upon submitting filing details|
+
+Configuration
+-------------
+System properties for psc-filing-api are defined in `application.properties`. These are normally configured per environment.
+
+Variable| Description                                                                                 | Example  |
+--------------------|---------------------------------------------------------------------------------|------------|
+MONGO_PSC_API_DB_NAME| The name of the collection responsible for storing PSC filings                              | collection_name |
+MONGODB_URL| The URL of the MongoDB instance where documents and application data should be stored       | mongodb://mongohost:27017|
```

Comment:
Please add a database name to the URL

Reply:
added `transacations_pscs` to config examples